contracts/modules/cover/Cover.sol

```diff
@@ -186,343 +185,185 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
   function buyCover(
     BuyCoverParams memory params,
     PoolAllocationRequest[] memory allocationRequests
```

Comment:
would rename this to poolAllocationRequests

The fact that there is another type called AllocationRequest makes it hard to disambiguate between what's coming from that array and what's being created inside the pool